package.json

```diff
@@ -48,6 +48,8 @@
     ]
   },
   "devDependencies": {
+    "@babel/core": "^7.18.9",
```

Comment:
Since you are adding new packages, please include the yarn.lock file, so everyone use the same package version.